app/assets/javascripts/models/beta/iteration.js

```diff
@@ -30,13 +44,13 @@ export const getIterationForStory = (story, project) => {
 };
 
 const createSprint = (sprintNumber = 0, startDate = 0, isFiller = false, velocity) => ({
-  number: sprintNumber + 1,
+  number: sprintNumber,
   startDate: startDate,
```

Comment:
We can improve it by a bit (now that our Babel supports it) using the shorthand notation:

```suggestion
  startDate,
```